src/components/QuoteList.js

```diff
@@ -0,0 +1,53 @@
+import { useEffect } from 'react';
+import getQuotes from '../services/quotes';
+import { useState } from 'react';
+import CharacterDropDown from './CharacterDropDown';
+import './List.css';
+export default function QuoteList() {
+  const [loading, setLoading] = useState(true);
+  const [quotes, setQuotes] = useState([]);
+  const [filteredQ, setFilteredQ] = useState([]);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const data = await getQuotes();
+      setQuotes(data);
+      setTimeout(() => {
+        setLoading(false);
+      }, 700);
```

Comment:
Don't forget to add this return () => {
          clearTimeout(timer);
        };
We would also need to set setTimeout as a const to prevent the error of an unmounted component being mounted improperly.